nucliadb/nucliadb/writer/api/v1/upload.py

```diff
@@ -489,7 +489,6 @@ async def upload(
     if rslug:
         path_rid = await get_rid_from_params_or_raise_error(kbid, slug=rslug)
 
-    logger.info(request.headers)
```

Comment:
I have deleted this after seeing some bearer tokens being logged in the terminal.

Please let me know if it has any use and needs to be put back